### PR TITLE
Add troubleshooting docs on prometheus OOM events

### PR DESCRIPTION
**Pull request by sgibson91**
fixes https://github.com/2i2c-org/infrastructure/issues/845

### CODE REVIEW

File: docs/topic/troubleshooting.md [hunk 1]
```diff
@@ -27,3 +29,25 @@ If you'd simply like to revert back to the state of the Kubernetes infrastructur
...
+  config:
+    prometheus:
+      server:
+        resources:
+          limits:
+            memory: {{RAM_LIMIT}}
+```
+
+The [default for this value is 2GB](https://github.com/2i2c-org/infrastructure/blob/dfe32345dc0b7d7cb961d6721bca291ab853e04b/support/values.yaml#L38).
+Try increasing this value in small increments and deploying using `pthon deployer deploy-support {{CLUSTER_NAME}}` until the server successfully starts up.
```
Comment (damianavila) [inline docs/topic/troubleshooting.md:52]:
```suggestion
Try increasing this value in small increments and deploying using `python deployer deploy-support {{CLUSTER_NAME}}` until the server successfully starts up.
```

**Review by choldgraf — APPROVED** [review]
looks good to me

**Review by damianavila — APPROVED** [review]
LGTM

**Comment by choldgraf** [comment]
two approvals so I am merging :-)

**Pull request merged**